### DISCUSSION TITLE
A route string problem

### DISCUSSION THREAD

**Discussion by shexm** [vikejs/vike#1224] [Discussion]
I use  route string likes '/a/@id' and '/b'.  When I navigated from '/b' to '/a/1' by the Client-side Router, it  scrolled to the top of the page.But when  I navigated from '/a/1' to '/a/2', it didn't work. And I have  infinite scroll components in '/a/1' and  '/a/2'. It triggers a bug when I scroll in  '/a/1' then navigate to '/a/2'. I don't know how to fix it.

**Comment by brillout** [comment]
Create a minimal reproduction, I'll then have a look at it.
